index.html

```diff
@@ -13,8 +13,8 @@
     <link href="./styles/external/bootstrap/justified-nav.css" rel="stylesheet">
     <link href="./styles/external/font-awesome/css/font-awesome.min.css" rel="stylesheet">
     <link href="./styles/external/tags-input/react-tagsinput.css" rel="stylesheet">
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <script src="./scripts/jquery.min.js"></script>
```

Comment:
I am not sure if it is better to separate them more, using different folders, or this way is ok too.
Also, maybe instead of "scripts" is better to rename the folder to "js"...